packages/fxa-payments-server/src/routes/Subscriptions/SubscriptionItem.tsx

```diff
@@ -50,24 +48,6 @@ export const SubscriptionItem = ({
 }: SubscriptionItemProps) => {
   const { locationReload } = useContext(AppContext);
 
-  if (!subscription) {
```

Comment:
This is where the error popped up during work on #5745. The scenario is that the fresh un-cached customer lists a subscription that the cached customer data doesn't include. But we never use the cached data anyway, so this error serves no purpose.